packages/salesforcedx-vscode-core/src/util/authInfo.ts

```diff
@@ -125,6 +125,10 @@ export class OrgAuthInfo {
       authInfo: await AuthInfo.create({ username })
     });
   }
+
+  public static async getOrgApiVersion() {
+    return await (await OrgAuthInfo.getConnection()).getApiVersion();
```

Comment:
total int: For a situation like this I'd probably go with 
```
const connection = await OrgAuthInto.getConnection();
const apiVersion = await connection.getApiVersion();
return apiVersion;
```
Granted it could be more concise, but I love me some code where I can set a breakpoint and see the result of an async call.